### ISSUE TITLE
Add releasing instructions

### ISSUE THREAD

**Issue by dlm** [compTAG/r-tda#19]

**Comment by jisuk1** [comment]
update new features in NEWS
update Version and Date in DESCRIPTION and man/TDA-package.Rd

**Comment by dlm** [comment]
Thanks!! After that, did you do the manual CRAN submission?

**Comment by jisuk1** [comment]
No, this was just listing the things to do before releasing new version.

I am planning to release TDA 1.6.3 to CRAN soon after few more testing on the code.

**Comment by dlm** [comment]
Cool.  Let me know when you have the list of steps complete so that we can write a release document.  There are a few tools that I found that can automate some of the processes (but we will have to see if what we do is similar enough).
